### PR TITLE
Fix typos in the spec

### PR DESCRIPTION
**Pull request by mattmcneeney**
Another simple one...

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -493,9 +493,9 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id?acc
...
+| 422 Unprocessable entity | MUST be returned if the requested change is not supported or if the request cannot currently be fulfilled due to the state of the instance (e.g. instance utilization is over the quota of the requested plan). Brokers SHOULD include a user-facing message in the body; for details see [Broker Errors](#broker-errors). Additionally, a `422 Unprocessable Entity` can also be returned if the broker only supports asynchronous update for the requested plan and the request did not include `?accepts_incomplete=true`; in this case the expected response body is: `{ "error": "AsyncRequired", "description": "This service plan requires client support for asynchronous service operations." }` |
```
Comment (duglin) [inline spec.md]: what's the change here?

Comment (mattmcneeney) [inline spec.md]: `Broker SHOULD` changed to `Brokers SHOULD`. Github didn't do well at highlighting the change did it!

Comment (mattmcneeney) [inline spec.md]: (and another reason to use 80 char lines!)

Comment (duglin) [inline spec.md]: thanks - and yes I can't wait to move to 80 chars

**Comment by duglin** [comment]
LGTM

**Comment by angarg12** [comment]
LGTM

**Comment by pmorie** [comment]
LGTM****

**Comment by avade** [comment]
LGTM

**Pull request merged**